modules/installer.nix

```diff
@@ -13,6 +13,11 @@ let cfg = config.nix-dabei; in
       type = types.attrs;
       default = import ../disk-layouts/zfs-simple.nix { inherit (cfg) diskDevice; };
     };
+    ntpServer = mkOption {
+      description = "NTP server to use for timesync during startup";
+      type = types.str;
+      default = "ptbtime1.ptb.de";
```

Comment:
Personal default since it's a stratum 1 that support NTP and NTS. Most NTP pools do not support NTS, making timesync one of the last MitM-able infrastructure protocols.

I'm not attached to them, happy to use upstream instead.

Reply:
Will fix in next iteration of this patch.